templates/dataset.html

```diff
@@ -651,12 +664,29 @@ <h3 class="modal-title" id="myModalLabel">Dataset name</h3>
         $(".csv_download_link").on("click", function(e) {
             /* Get the ID of the target clicked.*/
             let link_id = $(e.target).attr("id");
-            /* Prevent click propagation. */
-            e.stopPropagation();
             /* Get data associated with this id. */
             let data = $("#stats-panel").data(link_id);
             /* Call toCSV() on this element. */
             toCSV.apply(this, [data, link_id + ".csv"]);
         });
+        $("#datasetExportsForm").on("submit", function(e) {
+            /* Remove the alert if it exists */
+            $("#emailAlert").remove
+            /* Get the value of the email input element */
+            let form = $(e.currentTarget),
+                email = form.find("#emailInput").val();
+            /* Validate */
+            if (isEmail(email)) {
+                return;
+            }
+            else {
+                e.preventDefault();
+                /* Add an alert element if not already present. */
+                let emailAlert = $("#emailAlert");
+                if (emailAlert.length == 0) {
+                    form.append("<div class='alert alert-warning' role='alert' id='emailAlert'>Please enter a valid email address</div>");
```

Comment:
The alert is showing up with no space between it and the Generate button. Can some space be added? 

![Screen Shot 2021-01-25 at 8 56 32 AM](https://user-images.githubusercontent.com/1619369/105725578-1f4cec00-5ef7-11eb-97a7-b394545b23d8.png)